packages/cxl-ui/src/components/cxl-base-card.js

```diff
@@ -50,16 +54,19 @@ export class CXLBaseCardElement extends LitElement {
   }
 
   _renderHeaderName() {
-    return html` <div class="name">${this.name}</div> `;
+    return html`
+      <div class="name">
+        ${this.name}${this.completed
+          ? html`<vaadin-icon icon="lumo:checkmark"></vaadin-icon>`
+          : nothing}
+      </div>
+    `;
   }
 
   _renderHeaderAttributes() {
     return html`
       <div class="attributes">
-        <cxl-time
-          time=${this.time}
-          ?show-icon=${!!(this.theme.toLowerCase() === 'course')}
-        ></cxl-time>
+        <cxl-time time=${this.time} ?show-icon=${this.showTimeIcon}></cxl-time>
```

Comment:
Changing this logic impacts `<cxl-course-card>`. You set it ` @state() showTimeIcon = true;` in the component but it's different then it was originally. Please revisit original code and adjust